packages/codemods/src/codemods/v0.39.x/updateBabelConfig/updateBabelConfig.ts

```diff
@@ -0,0 +1,59 @@
+import fs from 'fs'
+import path from 'path'
+
+import getRWPaths from '../../../lib/getRWPaths'
+import prettify from '../../../lib/prettify'
+
+export const removeBabelConfig = async () => {
+  const rootBabelConfigPath = path.join(getRWPaths().base, 'babel.config.js')
+  const webBabelRcPath = path.join(getRWPaths().web.base, '.babelrc.js')
+
+  const webBabelConfigPath = path.join(getRWPaths().web.base, 'babel.config.js')
+
+  // Remove root babel config
+  if (fs.existsSync(rootBabelConfigPath)) {
+    const rootConfig = require(rootBabelConfigPath)
+    // If the rootConfig is the default, we can remove it
+    if (
+      Object.keys(rootConfig).length === 1 &&
+      rootConfig.presets[0] === '@redwoodjs/core/config/babel-preset'
+    ) {
+      console.log('Removing root babel.config.js')
+      fs.rmSync(rootBabelConfigPath)
+    } else {
+      // They have custom config in the root babel.config
+      // Fail and ask them to move config manually
+      console.warn('Detected custom config in your root babel.config.js')
+      throw new Error(
+        'Cannot automatically codemod your project. Please move your root babel.config.js settings manually'
+      )
+    }
+  }
+
+  if (fs.existsSync(webBabelRcPath)) {
+    const webConfig = require(webBabelRcPath)
+
+    // If its the default .babelrc.js
+    if (
+      Object.keys(webConfig).length === 1 &&
+      webConfig.extends === '../babel.config.js'
```

Comment:
Similar to another comment, ok to assume that if there's just one key, it's `extends`?

Reply:
This one should be OK, because if extends doesn't exist, it won't break!